src/com/zone5cloud/retrofit/core/OkHttpClientInterceptor_Authorization.java

```diff
@@ -138,30 +139,33 @@ private String getBaseUrl(HttpUrl url) {
 	public Response intercept(Chain chain) throws IOException {
 		Request originalRequest = chain.request();
 		String path = originalRequest.url().encodedPath();
+		String originalRequestUrl =  getBaseUrl(originalRequest.url());
 		Request.Builder builder = originalRequest.newBuilder();
 		
 		AuthToken token = this.token.get();
 		if (token != null && token.getBearer() != null && Endpoints.requiresAuth(path)) {
-			refreshIfRequired(chain, getBaseUrl(originalRequest.url()));
-			
+
+			refreshIfRequired(chain, originalRequestUrl);
+
```

Comment:
you need to base zone5BaseUrl to refreshIfRequired:

`refreshIfRequired(chain, zone5BaseUrl);`